docs/source/getting_started.rst

```diff
@@ -0,0 +1,39 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application. It is primarily used as a core component of BrightHive Data Trusts but has many :ref:`use-cases<use-cases>`. 
+
+Definitions
+-----------
+
+Data Trust
+    Legal, technical, and governance frameworks that enable networks of organizations to securely and responsibly share and collaborate with data, generating new insights and increasing their combined impact.
+
+Data Resource
+    Loosely defined as data that is owned by or stewarded over by members of Data Trusts. These are core elements of a Data Trust. In the context of the DRG, a Data Resource is a relational database with a RESTful API. 
```

Comment:
I think these sentences would be easier to read, if you remove some of the passive structures.

Let's try:

> The core element of a BrightHive Data Trust. Members of a Data Trust can own, manage, or oversee data resources. In the context of the DRG, a Data Resource is a relational database with a RESTful API, i.e., one instance of a DRG supports one data resource.

Reply:
Is my last sentence true ^^^?

Reply:
I think a data resource is intended to be one table and the related APIs for it so the DRG allows many data resources. But our definition of a data resource may not be perfect yet.